### PR TITLE
Fix resolution scaling for ESP

### PR DESCRIPTION
**Pull request by manianac**
ESP was glitching out on dual monitor setups or windowed modes versus fullscreen.

Tried a multitude of ideas, settled on DirectX window size for calculations

### CODE REVIEW

File: hooks/DirectX.cpp [hunk 1]
```diff
@@ -41,6 +41,14 @@ typedef struct Cache
 
 static cache_t s_Cache;
 
+ImVec2 DirectX::GetWindowSize()
```
Comment (Suwup) [inline hooks/DirectX.cpp:44]: This brakes windowed mode, prob bcs of the title bar and edges of the window.

File: gui/esp.hpp [hunk 2]
```diff
@@ -15,16 +15,17 @@ static bool IsWithinScreenBounds(Vector2& pos)
 
 static ImVec2 GetResolution()
 {
-	return Screen_get_fullScreen(nullptr) ? GetDesktopResolution() : ImVec2((float)Screen_get_width(nullptr), (float)Screen_get_height(nullptr));
+	return DirectX::GetWindowSize();
```
[the remark this answers is not in the document]
Comment (manianac) [reply gui/esp.hpp:18]: In case we would need to revert it.  Turns out it was necessary

File: hooks/DirectX.cpp (hunk 1 above)
Comment (manianac) [reply hooks/DirectX.cpp:44]: You were spot on, I have used your suggestion

**Review by Suwup — APPROVED** [review]
Works fine

**Pull request merged**